private-templates-service/src/config/passport.ts

```diff
@@ -0,0 +1,41 @@
+import passport from 'passport';
+import passportAD from 'passport-azure-ad';
+import config from './config';
+import logger from '../util/logger';
+
+enum LoggingLevels {
+    Info = 'info',
+    Warn = 'warn',
+    Error = 'error'
+}
+
+var options = {
+    identityMetadata: config.identityMetadata,
+    clientID: config.clientID,
+    validateIssuer: config.validateIssuer,
+    issuer: undefined,                                    // For tenant-specific endpoint, issuer from metadata is used by default
+    passReqToCallback: false,
+    isB2C: false,
+    policyName: undefined,
+    allowMultiAudiencesInToken: config.allowMultiAudiencesInToken,
+    audience: config.clientID,
+    loggingLevel: LoggingLevels.Info
+};
+
+var OIDCBearerStrategy = passportAD.BearerStrategy;
```

Comment:
Is there a reason for defining this seperately or can you make the next line passportAD.BearerStrategy(.......)?

Reply:
Thanks for the catch, fixed!